imod/tests/fixtures/backward_compatibility_fixture.py

```diff
@@ -8,8 +8,13 @@
 def imod5_dataset():
     tmp_path = imod.util.temporary_directory()
     data = imod.data.imod5_projectfile_data(tmp_path)
-    _load_imod5_data_in_memory(data[0])
-    return data[0]
+    data = data[0]
+    # Fix data for ibound  as it contains floating values like 0.34, 0.25 etc.
+    ibound = data["bnd"]["ibound"]
+    ibound = ibound.where(ibound <= 0, 1)
+    data["bnd"]["ibound"] = ibound
+    _load_imod5_data_in_memory(data)
+    return data
```

Comment:
This cleanup also occurs in the tests ``test_mf6_dis.py``, please remove the ibound cleanup from these tests.

Also: You might get some increased performance by first loading the data into memory and then cleaning up ibound.